src/sst/core/model/python/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+# SST-CORE src/sst/core/model/python CMake
+#
+# Copyright 2009-2021 National Technology and Engineering Solutions of Sandia,
+# LLC (NTESS).  Under the terms of Contract DE-NA-0003525, the U.S. Government
+# retains certain rights in this software.
+#
+# See LICENSE for full license details
+#
+
+add_library(
+  modelpython OBJECT
+  pymodel.cc pymodel_link.cc pymodel_comp.cc pymodel_unitalgebra.cc
+  pymodel_stat.cc pymodel_statgroup.cc)
+
+
+target_include_directories(modelpython PUBLIC ${sst-core_SOURCE_DIR}/../src/)
```

Comment:
These will be used in downstream projects. You will want to use BUILD/INSTALL interface.